### PR TITLE
feat(alert, chip, date-picker, input-date-picker, filter, pagination, pick-list-item, rating): capitalize default text values

### PR DESCRIPTION
**Pull request by benelan**
**Related Issue:** #2534

## Summary
- Capitalize the first letter of the TEXT string values in the resource files
- Capital the first letter of the ``@default`` JSDoc tags in the component .tsx files

### CODE REVIEW

**Review by driskull — APPROVED** [review]
👍 💯

File: src/components/calcite-rating/calcite-rating-resources.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export const TEXT = {
   rating: "Rating",
-  stars: "stars: ${num}"
+  stars: "Stars: ${num}"
```
Comment (caripizza) [inline src/components/calcite-rating/calcite-rating-resources.ts:3]: Can you update the `@default` JSDoc string in rating.tsx too?
https://github.com/Esri/calcite-components/blob/master/src/components/calcite-rating/calcite-rating.tsx#L67

**Review by caripizza — APPROVED** [review]
Thanks @benelan! Just one last JSDoc tag straggler in calcite-rating.tsx otherwise looks great (we're displaying the docs-json data via this tag over in dev docs) 👍

**Comment by benelan** [comment]
Nice catch @caripizza

**Pull request merged**